app/models/non_hmis_assessment.rb

```diff
@@ -76,6 +78,10 @@ def self.known_assessments_for_matching
       merge(IdentifiedPathwaysVersionThree.new(assessment_type: :transfer_assessment).for_matching).
       merge(DeidentifiedPathwaysVersionThree.new(assessment_type: :pathways_2021).for_matching).
       merge(DeidentifiedPathwaysVersionThree.new(assessment_type: :transfer_assessment).for_matching).
+      merge(IdentifiedPathwaysVersionFour.new(assessment_type: :pathways_2024).for_matching).
+      merge(IdentifiedPathwaysVersionFour.new(assessment_type: :transfer_assessment).for_matching).
```

Comment:
Hmm, we don't have a new transfer assessment yet right?  I think we probably don't need these here, but I could be convinced otherwise.

Reply:
I'll pull out the V4 transfer_assessment lines.